messages/plugin.generator.md

```diff
@@ -4,32 +4,32 @@ Time to build an sf plugin! Version %s
 
 # question.internal
 
-Are you building a plugin for an internal Salesforce team
+Are you building a plugin for an internal Salesforce team?
```

Comment:
The inquirer library puts `?` at the beginning of the question like this:

```
? Are you building a plugin for an internal Salesforce team (Y/n)
```

Do you think it'll look odd to have a ? on both ends?

```
? Are you building a plugin for an internal Salesforce team? (Y/n)
```

Reply:
Good point.  But I think it's better to keep the two question marks.   The first one is a different color (in my terminal at least) and acts as a "this line is some kind of prompt" identifier, as opposed to the ">" character, for example, which identifies an option.  The verbiage itself is in bold, and without the ending question mark it looks incomplete and a mistake, IMHO.

We also followed this format (ending question mark) w/Shane's commands to generate objects/fields/etc, so it's nice to be consistent. 

So I'd prefer to keep the ending question mark.